code/controllers/subsystems/job.dm

```diff
@@ -341,6 +341,8 @@
 		if(istype(S, /obj/effect/landmark/start) && istype(S.loc, /turf))
 			H.forceMove(S.loc)
 			H.lastarea = get_area(H.loc)
+			H.lastarea.lightswitch = TRUE
+			H.lastarea.set_lightswitch(TRUE)
```

Comment:
This is just, really bad design. If you have a function that's called `set_x` which accepts a new value, then by god that function should actually set the variable it's describing.

So basically: the only thing you should need in this line is `H.lastarea.set_lightswitch(TRUE)`.

Reply:
i have no idea why i did this